products/compute/terraform.yaml

```diff
@@ -12,6 +12,28 @@
 # limitations under the License.
 
 --- !ruby/object:Provider::Terraform::Config
+overrides:
+  - !ruby/object:Provider::Terraform::Override
+    name: 'HttpHealthCheck'
```

Comment:
@nelsonjr 

Another option would be to use a Hash for the overrides instead of an array.
The key in the hash would be used to match with the resource name in api.yaml.
If we want to use a different name in Terraform, we could set the name to a different value.

For instance we could have:
```yaml
overrides:
  TargetVpnGateway:
    name: VpnGateway
```

Reply:
Right on, I was going to suggest to move away from array for another reason too: because this hash also allows us to do the following:

```
overrides: !ruby/object:Api::Overrides
  TargetVpnGateway: !ruby/object:Api::Resource   # or Terraform::Resource
   ... here you can define/redefine anything you wish for the resource, using the resource itself 
   (or one of its descendants).
```